ontology.ttl

```diff
@@ -255,8 +255,9 @@ owl:versionIRI rdf:type owl:AnnotationProperty .
          rdfs:subClassOf :WorkflowParameter ,
                          [ rdf:type owl:Restriction ;
                            owl:onProperty :value ;
-                           owl:allValuesFrom xsd:boolean
-                         ] .
+                           owl:allValuesFrom xsd:boolean, xsd:bool
+                         ]  ;
+       rdfs:sameAs xsd:boolean, xsd:bool .
```

Comment:
I consulted the OWL specification. owl:sameAs is used for individuals, for classes there is owl:equivalentClass 
Let's use it, probably

Reply:
`equivalentClass` is now there. Does it make more sense ?